events/wmt21.md

```diff
@@ -209,3 +209,32 @@ Thursday, November 11, 2021
 | 15:00 – 15:15 | [***Subword Mapping and Anchoring across Languages***](https://arxiv.org/pdf/2109.04556.pdf) <br>Giorgos Vernikos and Andrei Popescu-Belis |
 | 15:15 – 15:30 | [***Uncertainty-Aware Machine Translation Evaluation***](https://arxiv.org/pdf/2109.06352.pdf) <br>Taisiya Glushkova, Chrysoula Zerva, Ricardo Rei and André F. T. Martins |
 | 15:30 – 15:45 | [***Sometimes We Want Ungrammatical Translations***](https://aclanthology.org/2021.findings-emnlp.275.pdf) <br>Prasanna Parasarathi |
+
+## Results
+
+### Shared Task: Machine Translation of News
+
+#### Translation into-English (SR+DC)
+*Systems ordered by DA score z-score; systems within a cluster are considered tied; lines indicate clusters according to Wilcoxon rank-sum test p < 0.05*
+
+|     |     |     |     |     |
+| --- | --- | --- | --- | --- |
+| **Language pair** | **Rank** | **Ave.** | **Ave. z** | **System** |
```

Comment:
What does Rank mean here?

Reply:
It's the change in the order of systems, but as I think about it, it may not be valuable without information on clusters, so maybe I should remove it from the table. What do you think?